features/admin.users.v1/components/user-profile/user-profile-form.tsx

```diff
@@ -124,6 +126,7 @@ const UserProfileForm: FunctionComponent<UserProfileFormPropsInterface> = ({
     const attributeDialectsFeatureConfig: FeatureAccessConfigInterface = useSelector(
         (state: AppState) => state.config.ui.features?.attributeDialects
     );
+    const featureConfig: FeatureConfigInterface = useSelector((state: AppState) => state?.config?.ui?.features);
```

Comment:
Shall we get the users feature config itself rather than the whole feature config? Ex: line 126